packages/tailwind/src/tailwind.tsx

```diff
@@ -69,19 +69,20 @@ export const Tailwind: React.FC<TailwindProps> = ({ children, config }) => {
           }
 
           if (domNode.attribs?.class) {
-            if (hasResponsiveStyles) {
-              domNode.attribs.class = domNode.attribs.class.replace(
-                /[:#\!\-[\]\/\.%]+/g,
-                "_"
-              );
-            } else {
-              const currentStyles = domNode.attribs.style
-                ? `${domNode.attribs.style};`
-                : "";
-              const tailwindStyles = twi(domNode.attribs.class);
-              domNode.attribs.style = `${currentStyles} ${tailwindStyles}`;
-              delete domNode.attribs.class;
-            }
+            const currentStyles = domNode.attribs.style ? `${domNode.attribs.style};` : ""
+            const tailwindStyles = twi(domNode.attribs.class)
+            domNode.attribs.style = `${currentStyles} ${tailwindStyles}`
+            console.log("class", domNode.attribs.class)
+            domNode.attribs.class = domNode.attribs.class
+            // remove all non-responsive classes (ex: m-2 md:m-4 > md:m-4)
+            .split(" ")
+            .filter((className) => className.search(/^.{2}:/) !== -1)
+            .join(" ")
+            // replace all non-alphanumeric characters with underscores
+            .replace(/[:#\!\-[\]\/\.%]+/g, "_")
+
+            console.log("new class", domNode.attribs.class)
```

Comment:
log

Reply:
also thank you 🤦

Reply:
No problem, im seeing this right now and its a real pain. Would love to get this in

Reply:
Yeah, I've yalc'd it for our service currently because we need it 😅